tests/test_api.rs

```diff
@@ -72,6 +72,27 @@ fn handle_scope_numbers() {
   }
 }
 
+// TODO: the type checker is kumbaya with this but in reality the
+// `Local<Integer>` created at the end of the test is created in HandleScope
+// `hs2` and not in `hs1` as specified. When this local is accessed, which is
+// after `hs2` is destroyed, a crash happens.
+// #[test]
+// fn handle_scope_early_drop() {
```

Comment:
In the future use #[ignore] so we continue checking that it compiles until it can be fixed

Reply:
Makes sense, thanks.
I had no idea that existed.